tests/ci/integration/tpm2_tss_patch/aws-lc-tpm2-tss.patch

```diff
@@ -0,0 +1,122 @@
+From 582d4fe349c89fd2cfbd66957c472b7c687eb429 Mon Sep 17 00:00:00 2001
+From: Justin W Smith <103147162+justsmth@users.noreply.github.com>
+Date: Fri, 15 Dec 2023 15:27:22 -0500
+Subject: [PATCH] Minor fixes for AWS-LC
+
+---
+ src/tss2-esys/esys_crypto_ossl.c   | 4 +++-
+ src/tss2-fapi/ifapi_curl.c         | 6 +++---
+ src/tss2-fapi/ifapi_get_web_cert.c | 1 +
+ test/unit/fapi-eventlog.c          | 8 +++++---
+ 4 files changed, 12 insertions(+), 7 deletions(-)
+
+diff --git a/src/tss2-esys/esys_crypto_ossl.c b/src/tss2-esys/esys_crypto_ossl.c
+index 1620788c..2924638a 100644
+--- a/src/tss2-esys/esys_crypto_ossl.c
++++ b/src/tss2-esys/esys_crypto_ossl.c
+@@ -8,6 +8,8 @@
+ #include <config.h>
+ #endif
+ 
++#include <openssl/mem.h>
++#include <openssl/bn.h>
```

Comment:
What was missing/in the wrong header without this?

Reply:
Could other projects hit the same issue? Does it make sense to "fix" our headers like https://github.com/aws/aws-lc/blob/main/include/openssl/evp.h#L65